api/integrations/sentry/tests/test_sampler.py

```diff
@@ -0,0 +1,38 @@
+from django.conf import settings
+
+from integrations.sentry.samplers import block_health_sampler
+
+
+def test_block_health_sampler_empty_context_returns_defaut_sample_rate():
+    # Given
+    settings.SENTRY_TRACE_SAMPLE_RATE = 10
```

Comment:
It's better to use the `@override_settings()` decorator to do this so your tests become something like the following: 

```python
sample_rate = 10

@override_settings(SENTRY_TRACE_SAMPLE_RATE=sample_rate)
def test_block_health_sampler_empty_context_returns_defaut_sample_rate():
    assert block_health_sampler({}) == sample_rate
```

See documentation here: https://docs.djangoproject.com/en/3.2/topics/testing/tools/#django.test.override_settings